.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: Reusable action of Integration tests
+
+on: 
+  workflow_call:
+    secrets:
+      CI_USER_TOKEN:
+        required: true
+      TRAVIS_COM_TOKEN:
+        required: true
+      PREP:
+        required: false
+      RELEASE:
+        required: false
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        # You should create a personal access token and store it in your repository
+        token: ${{ secrets.CI_USER_TOKEN }}
+        repository: 'optimizely/travisci-tools'
+        path: 'home/runner/travisci-tools'
+        ref: 'master'
+    - name: set SDK Branch if PR
+      if: ${{ github.event_name == 'pull_request' }}
+      run: |
+        echo "SDK_BRANCH=${{ github.head_ref }}" >> $GITHUB_ENV
+        echo "TRAVIS_BRANCH=${{ github.base_ref }}" >> $GITHUB_ENV
+    - name: set SDK Branch if not pull request
+      if: ${{ github.event_name != 'pull_request' }}
+      run: |
+        echo "SDK_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
+        echo "TRAVIS_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
```

Comment:
these variables doesn't seem correct to set with refname.

Reply:
`ref_name` has the `The branch or tag name that triggered the workflow run.` as per the official documentation, in case build was not triggered by pull request. This is required by `travisci-tools/trigger-script-with-status-update.sh`.

Reply:
remove it, we don't have any relation with the current repo to `trigger-script-with-status-update.sh`